test/utils/actions.ts

```diff
@@ -1,395 +1,201 @@
+    await SettingsScreen.deleteAccountsButton.click();
+    await ConfirmDeleteAccountsScreen.confirmPhraseInput.setValue(
+      'DELETE ALL ACCOUNTS'
     );
+    await ConfirmDeleteAccountsScreen.deleteAllButton.click();
 
-    await resetConfirmButton.click();
-    await this.driver.wait(until.stalenessOf(resetConfirmButton), this.wait);
-  },
-
-  async open(this: mocha.Context) {
-    await this.driver.get(this.extensionUrl);
+    expect(await GetStartedScreen.getStartedButton).toBeDisplayed();
   },
 
-  async closeBgTabs(this: mocha.Context, foreground: string) {
-    for (const handle of await this.driver.getAllWindowHandles()) {
+  closeBgTabs: async (foreground: string) => {
+    for (const handle of await browser.getWindowHandles()) {
       if (handle !== foreground) {
-        await this.driver.switchTo().window(handle);
-        await this.driver.close();
+        await browser.switchToWindow(handle);
+        await browser.closeWindow();
       }
     }
-    await this.driver.switchTo().window(foreground);
+
+    await browser.switchToWindow(foreground);
   },
 };
 
-export const Assets = {
-  async addAccount(this: mocha.Context) {
-    await this.driver.sleep(DEFAULT_ANIMATION_DELAY);
-
-    await this.driver
-      .wait(
-        until.elementLocated(By.css('[data-testid="otherAccountsButton"]')),
-        this.wait
-      )
-      .click();
-
-    await this.driver
-      .wait(
-        until.elementLocated(By.css('[data-testid="addAccountButton"]')),
-        this.wait
-      )
-      .click();
+export const PopupHome = {
+  addAccount: async () => {
+    await HomeScreen.otherAccountsButton.click();
+    await OtherAccountsScreen.addAccountButton.click();
   },
-  async getActiveAccountName(this: mocha.Context) {
-    return this.driver
-      .wait(
-        until.elementLocated(
-          By.css(
-            '[data-testid="activeAccountCard"] [data-testid="accountName"]'
-          )
-        ),
-        this.wait,
-        'Could not get active account name'
-      )
-      .getText();
+
+  getActiveAccountName: async () => {
+    const accountName = HomeScreen.activeAccountName;
+    await accountName.waitForDisplayed();
+    return await accountName.getText();
   },
-  async getOtherAccountNames(this: mocha.Context) {
-    await this.driver
-      .wait(
-        until.elementLocated(By.css('[data-testid="otherAccountsButton"]')),
-        this.wait
-      )
-      .click();
-
-    const accountNameElements = await this.driver
-      .wait(
-        until.elementsLocated(
-          By.css('[data-testid="accountCard"] [data-testid="accountName"]')
-        ),
-        500
-      )
-      .catch((): WebElement[] => []);
-
-    const accountNames = await Promise.all(
-      accountNameElements.map(accName => accName.getText())
-    );
 
-    await this.driver.findElement(By.css('div.arrow-back-icon')).click();
+  getOtherAccountNames: async () => {
+    await $('[data-testid="otherAccountsButton"]').click();
```

Comment:
Raw selectors? I'm pretty sure there's a helper for that

Reply:
done